ReadMe.md

```diff
@@ -179,7 +179,7 @@ gcs:
   storage_class: STANDARD      # GCS_STORAGE_CLASS
 
   # GCS_OBJECT_LABELS, allow setup metadata for each object during upload, use {macro_name} from system.macros and {backupName} for current backup name
-  # for environment use following format "key1:value1,key2:value2" 
+  # Use the following format "key1:value1,key2:value2" 
```

Comment:
need another text here, 
`for environment` here means for environment variable GCS_OBJECT_LABELS
for config file you shall continue to use map in YAML